ui/index.js

```diff
@@ -72,6 +78,9 @@ export default function launchMetamaskUi(opts, cb) {
       );
       return;
     }
+
+    endTrace({ name: 'Get State' });
```

Comment:
if `backgroundConnection.getState(` fails do we need to end the trace? Or maybe just delete the trace from the internal `trace.ts` `Map`?

Reply:
Good spot, moved to always end, even in case of error.